ChangeLog.d/aes-perf.txt

```diff
@@ -1,4 +1,4 @@
 Features
-   * AES performance improvements on 64-bit architectures. Uplift
-     varies by platform, toolchain, optimisation flags and mode,
-     in the 0 - 84% range. Aarch64, gcc and GCM/XTS benefit the most.
+   * AES performance improvements. Uplift varies by platform,
+     toolchain, optimisation flags and mode, up to 4.5x.
```

Comment:
if it's ~5% for most, and 4.5x for XTS, should we be more specific (to manage expectations)?

Reply:
Probably, XTS is a bit of an edge-case really. I'll improve the wording.